src/utils/math_utils.cairo

```diff
@@ -55,4 +57,30 @@ mod MathUtils {
             a - b
         }
     }
+
+    /// TODO: Replace when the implementation of integers in Cairo is released. 
+    /// @notice This function is an override for Orion's integer division. 
+    /// This is necessary because the behavior for the division of negative 
+    /// numbers works incorrectly when it comes to rounding.
+    fn i32_div(a: i32, b: i32) -> i32 {
+        assert(b.mag != 0, 'b can not be 0');
```

Comment:
could you add a test case for this assert?

Reply:
Done!